src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/Extensions/INotifyPropertyChangedEx.shared.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Xamarin.CommunityToolkit.ObjectModel.Extensions
+{
+	public static class INotifyPropertyChangedEx
+	{
+		public static void WeakSubscribe<T>(this INotifyPropertyChanged target, T subscriber, Action<T, object, EventArgs> action)
```

Comment:
I just love it! 
@brminnick do you think that we can add this helper and others (described [here](https://faithlife.codes/blog/2008/08/event_subscription_using_weak_references/) into your WeakEventManager? I think they could be very helpful for devs!

Reply:
Glad you liked it, I had lots of fun creating it =)
This is already a public extension method, so users would be able to use it on anything that implements `INotifyPropertyChanged` (that's why I added it as an extension method instead of member of `LocalizationResourceManager`)

Reply:
Yeah, I saw that is public for everyone to use it <3. But I want to see if we can improve our WeakEvent manager implementation and also bring the other methods